subsys/bluetooth/host/conn.c

```diff
@@ -1936,6 +1944,17 @@ int bt_conn_get_info(const struct bt_conn *conn, struct bt_conn_info *info)
 	case BT_CONN_TYPE_BR:
 		info->br.dst = &conn->br.dst;
 		return 0;
+#endif
+#if defined(CONFIG_BT_ISO)
+	case BT_CONN_TYPE_ISO:
+		if (!conn->iso.is_bis) {
+			info->le.dst = &conn->iso.acl->le.dst;
+			info->le.src = &bt_dev.id_addr[conn->iso.acl->id];
```

Comment:
Can this function be called on a ISO bt_conn where the ACL isn't established? If so, we need a check for `conn->acl`.

Reply:
CIS always shall be bound to an ACL otherwise it cannot exist.